client/web/src/global/CoolCodeIntel.tsx

```diff
@@ -180,18 +182,64 @@ export const ReferencesList: React.FunctionComponent<
         token: Token
     }
 > = props => {
-    const [activeLocation, setActiveLocation] = useState<Location>()
     const [filter, setFilter] = useState<string>()
     const debouncedFilter = useDebounce(filter, 150)
 
+    const {
+        lsifData,
+        error,
+        loading,
+        referencesHasNextPage,
+        implementationsHasNextPage,
+        fetchMoreReferences,
+        fetchMoreImplementations,
+        fetchMoreReferencesLoading,
+        fetchMoreImplementationsLoading,
+    } = usePreciseCodeIntel({
+        variables: {
+            repository: props.token.repoName,
+            commit: props.token.commitID,
+            path: props.token.filePath,
+            // On the backend the line/character are 0-indexed, but what we
+            // get from hoverifier is 1-indexed.
+            line: props.token.line - 1,
+            character: props.token.character - 1,
+            filter: filter || null,
```

Comment:
```suggestion
            filter: debouncedFilter || null,
```

Right? We don't want to spam network requests

Reply:
Yes! Thanks for this, I just noticed that something was off.